src/levanter/eval.py

```diff
@@ -123,6 +125,17 @@ def eval_callback(step: StepInfo):
             _join_prefix(prefix, "loading_time"): result.total_eval_loading_time,
             _join_prefix(prefix, "total_time"): time_fn(),
         }
+        if (gate_hist := result.extras.get("gate_hist", None)) is not None:
```

Comment:
so i think i'm gonna have a strong preference for 

1) extracting this block (and the part in the loop) into a class (sort of like runningmean)
2) not actually checking the usage of it in taggedevaluator (or in the models) into main, but instead
3) making a little guide on how to add it in, since it's something that people want to play with sometimes but kinda adds a bunch of noise